package_manifest.yaml

```diff
@@ -312,9 +312,11 @@ plugins_packages:
   vars:
     package_base_dir: 'packages/plugins/'
     releasers:
-    - koji-foreman-plugins
-  hosts:
+    - koji-foreman-plugin
```

Comment:
Could you revert this change? This breaks obal

Reply:
yes